src/main/java/com/razorpay/RazorpayClient.java

```diff
@@ -8,6 +8,10 @@
   public RefundClient Refunds;
   public OrderClient Orders;
   public InvoiceClient Invoices;
+  public TokenClient Tokens;
```

Comment:
It doesn't make sense to expose TokenClient as tokens are linked with a customer entity. You can move this to the CustomerClient constructor. @ankit-rzp Need your input on this